client/src/api/fakeAuthApi.js

```diff
@@ -0,0 +1,39 @@
+export const login = (email, password, users) => {
+
+    const findUser = (email) => {
+        return users.find( user => user.email === email)
```

Comment:
Removing return and curly braces will work just fine here.You should remove them